testsuite/pytests/test_stdp_synapse.py

```diff
@@ -63,20 +65,20 @@ def init_params(self):
             "Wmax": 15.0,
             "weight": self.init_weight,
         }
-        self.neuron_parameters = {"tau_minus": self.tau_post, "t_ref": 1.0}
-
-        # While the random sequences, fairly long, would supposedly
-        # reveal small differences in the weight change between NEST
-        # and ours, some low-probability events (say, coinciding
-        # spikes) can well not have occurred. To generate and
-        # test every possible combination of pre/post order, we
-        # append some hardcoded spike sequences
+        self.neuron_parameters = {
+            "tau_minus": self.tau_post,
+        }
+
+        # While the random sequences, fairly long, would supposedly reveal small differences in the weight change
+        # between NEST and ours, some low-probability events (say, coinciding spikes) can well not have occurred.
+        # To generate and test every possible combination of pre/post order, we append some hardcoded spike sequences.
         self.hardcoded_pre_times = np.array([1, 5, 6, 7, 9, 11, 12, 13, 14.5, 16.1], dtype=float)
-        self.hardcoded_post_times = np.array([2, 3, 4, 8, 9, 10, 12, 13.2, 15.1, 16.4], dtype=float)
-        self.hardcoded_trains_length = 2.0 + max(np.amax(self.hardcoded_pre_times), np.amax(self.hardcoded_post_times))
+        self.hardcoded_post_times = np.array([2, 3, 4, 8, 9, 10, 12, 12.2, 14.1, 15.4], dtype=float)
```

Comment:
Does the test still pass on the old spike times? Can we run the test on both sequences?

Reply:
Sure, I can include the previous spike trains as well